### PR TITLE
Add context menu item for Audio resources.

### PR DESCRIPTION
**Pull request by Deledrius**
Restores the View Audio context menu item.

### CODE REVIEW

File: viewImageContextMenuItem/options.html [hunk 1]
```diff
@@ -12,6 +12,10 @@
 <body>
 
 <form class="panel-section panel-section-formElements" id="options">
+  <div class="browser-style section-option">
+    <input type="checkbox" id="show-view-audio" checked="checked" />
+    <label for="show-view-image">Show View Audio</label>
```
Comment (agoupinets) [inline viewImageContextMenuItem/options.html]:
```suggestion
    <label for="show-view-audio">Show View Audio</label>
```

File: viewImageContextMenuItem/options.js [hunk 2]
```diff
@@ -24,6 +26,7 @@ function saveOptions(event) {
   }
 
   const options = {
+    "show-view-image": document.querySelector("#show-view-audio").checked,
```
Comment (agoupinets) [inline viewImageContextMenuItem/options.js]:
```suggestion
    "show-view-audio": document.querySelector("#show-view-audio").checked,
```

File: viewImageContextMenuItem/options.js [hunk 3]
```diff
@@ -32,6 +35,7 @@ function saveOptions(event) {
     "shift-left-click-action": document.querySelector("#shift-left-click-action").value,
     "ctrl-shift-left-click-action": document.querySelector("#ctrl-shift-left-click-action").value,
     "middle-click-action": document.querySelector("#middle-click-action").value,
+    "action-key-view-image": cleanActionKey(document.querySelector("#action-key-view-audio").value),
```
Comment (agoupinets) [inline viewImageContextMenuItem/options.js]:
```suggestion
    "action-key-view-audio": cleanActionKey(document.querySelector("#action-key-view-audio").value),
```

**Review by agoupinets — APPROVED** [review]
LGTM, with a couple name corrections

**Comment by Deledrius** [comment]
Whoops, sorry about those few typos.  Thought I caught all of them. 😁  Thanks for accepting it!

**Pull request merged**